docs/api-configuration.rst

```diff
@@ -37,55 +40,67 @@ When specifying an annotation, your proto will need to import the file
 where the annotation is defined. If you try to use an annotation without
 importing the dependency proto, then ``protoc`` will give you an error.
 
-All of the protos discussed here are in the `googleapis`_ repository,
-on the ``annotated`` branch, and they are consistently in the
-``google.api.experimental`` package. While this remains experimental,
-the best course is probably to clone the repository:
+These protos live on the ``input-contract`` branch in the
+`api-common-protos`_ repository.
+
+Your best bet is to likely clone this repository:
 
 .. code-block:: shell
 
-  $ git clone git@github.com:googleapis/googleapis.git
-  $ cd googleapis
-  $ git checkout --track -b annotated origin/annotated
+  $ git clone git@github.com:googleapis/api-common-protos.git
+  $ cd api-common-protos
+  $ git checkout --track -b input-contract origin/input-contract
 
 Once this is done, you will need to specify the root of this repository
 on disk as a ``--proto_path`` whenever invoking ``protoc``.
 
-.. _googleapis: https://github.com/googleapis/googleapis/
+.. _api-common-protos: https://github.com/googleapis/api-common-protos/tree/input-contract
+
 
 
 API Client Information
 ~~~~~~~~~~~~~~~~~~~~~~
 
 The most important piece of information this plugin requires is information
-about the client library itself: what should it be called, how is it licensed,
-and so on.
+about the client library itself: what should it be called, what is its proper
+namespace, and so on.
 
-This is rolled up into a strcuture spelled ``Client``, and the annotation
-is defined in `google/api/experimental/client.proto`_.
+This is rolled up into a strcuture spelled ``Metadata``, and the annotation
+is defined in `google/api/metadata.proto`_.
```

Comment:
I think this sentence needs rewording.

I am mainly unsure about using the word `spelled`.